### ISSUE TITLE
pscp not working on V.1.4.0.9

### ISSUE THREAD

**Issue by CollapsedMetal** [jimradford/superputty#727]
**Describe the bug**
On connecting via "File Transfer Option" it request for login credentials over and over again...

**To Reproduce**
1. Go to 'Right Click -> File Transfer'
2. It Request login credentials over and over again...

**Expected behavior**
It should connect right away.

**Additional notes**
**IT WORKS ON PREVIOUS VERSION V.1.4.0.8**

**Comment by yaoyao-cake** [comment]
YES ， Me too. And I use ssh_key(passphrase) to login my server with pscp.

**Comment by davidbenucci** [comment]
Confirmed. It asks always for passwords.

**Comment by Chealer** [comment]
I never used 1.4.0.8 so cannot confirm that this is a regression, but I also experience this, visibly with any session.

Is anyone successfully using file transfer on 1.4.0.9?

Thank you for reporting this @CollapsedMetal . Would you mind retitling this ticket to something like "File Transfer sessions fail to open (continuously prompt for credentials)"?

**Comment by Chealer** [comment]
I never used File Transfer. But if I understand correctly what it offers, an alternative is to use WinSCP, which is somewhat integrated in SuperPuTTY.

WinSCP is a free software application which needs to be installed separately, and SuperPuTTY needs to have its path configured.

**Comment by CollapsedMetal** [comment]
@Chealer issue title changed

**Comment by raul1ro** [comment]
I am using the v1.5 (available only x32) and I have the same problem.
The location of putty.exe is at putty x64 (same with pscp.exe location), if this is important

**Comment by nukupanda** [comment]
> **Describe the bug** On connecting via "File Transfer Option" it request for login credentials over and over again...
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. Go to 'Right Click -> File Transfer'
> 2. It Request login credentials over and over again...
> 
> **Expected behavior** It should connect right away.
> 
> **Additional notes** **IT WORKS ON PREVIOUS VERSION V.1.4.0.8**
> 
> Not similar to #717

select "Options-->GUI-->Security-->Allow plain text passwords on putty command line", in this way, the file transfer can be successfully connected.(It's translated by software, haha)
